src/test/java/com/xebialabs/xlt/ci/server/XLTestServerImplTest.java

```diff
@@ -1,65 +1,93 @@
 package com.xebialabs.xlt.ci.server;
```

Comment:
Any particular technical reason to switch to junit ?

I'd have appreciated it in a different pull.